modules/admin_manual/pages/configuration/files/external_storage/configuration.adoc

```diff
@@ -127,3 +129,7 @@ Alternatively, replace `–all` with the user name to trigger a rescan of the us
 for example every 15 minutes, which includes the mounted external storage.
 
 TIP: See xref:configuration/server/occ_command.adoc#the-filesscan-command[the occ’s file operations] for more information.
+
+== Known limitations
+
+- Removal of the external storage or change of its configuration does not remove metadata entries belonging to previous storage configuration. This may impact performance of the installation as previous configuration metadata entries get orphaned. Removal of orphaned entries requires manual deletion of orhpaned storage cache by its storage id. 
```

Comment:
```suggestion
- Removal of the external storage or change of its configuration does not remove metadata entries belonging to the previous storage configuration. This may impact performance of the installation as previous configuration metadata entries get orphaned. Removal of orphaned entries requires manual deletion of orphaned storage cache by its storage id. 
```